components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -0,0 +1,92 @@
+import Button from "components/atoms/Button/button";
+import { ChangeEvent, useEffect, useRef, useState } from "react";
+
+const HighlightInputForm = (): JSX.Element => {
+  const [isDivFocused, setIsDivFocused] = useState<boolean>(false);
```

Comment:
The explicit types aren't needed because its inferred by the provided value

```suggestion
  const [isDivFocused, setIsDivFocused] = useState(false);
```